### PR TITLE
Increase QEMURAM in container-host due to k3s

### PR DESCRIPTION
**Pull request by pdostal**
* Related to os-autoinst/os-autoinst-distri-opensuse#16911

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]
LGTM

File: job_groups/opensuse_tumbleweed.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ scenarios:
           machine: 64bit
       - microos_textmode
       - container-host:
-          machine: 64bit
+          machine: 64bit-4G-HD40G
           settings:
```
Comment (DimStar77) [inline job_groups/opensuse_tumbleweed.yaml:79]: jump from 1G to 4G just for k3s? 

Then k3s should probably be tested separately from the standard tests; having the default system run on low-mem and seeing that 'normal machines' actually have space for workloads is a good thing. With this setup we'll never catch that the base system now occupies 3.5G and leaving only little left to the actual users.

Comment (grisu48) [reply job_groups/opensuse_tumbleweed.yaml:79]: I tend to agree with DimStar here. 1GiB is the [minimal system configuration suggested for MicroOS](http://get.opensuse.org/microos/), so we should also test it with 1 GiB.

Comment (pdostal) [reply job_groups/opensuse_tumbleweed.yaml:79]: Maybe we can after skip the `k3s` part in MicroOS and just test it in Tumbleweed? And then even more later we may have separate `k4s` test suite.

**Pull request closed without merging**